.github/workflows/tox.yml

```diff
@@ -19,7 +19,9 @@ jobs:
     name: ${{ matrix.tox_env }}
     # macos is the only gh action platform with support for vagrant/virtualbox
     # https://github.com/actions/virtual-environments/issues/433
-    runs-on: macos-latest
+    # Keep 10.15 for now as there's no vagrant in macosx-latest
+    # ref: https://github.com/actions/virtual-environments/issues/4060
```

Comment:
Ideally we need a link to a ticket specific to the missing vagrant on 10.11 not the generic one, or it will never be fixed.